pkg/manager/manager.go

```diff
@@ -63,7 +63,20 @@ func NewManager(
 	log := log.FromContext(ctx)
 	log.Info("Creating manager")
 
-	narrowCache := narrowcache.NewConfig(kcfg, narrowcache.ConfigMaps, narrowcache.Secrets)
+	narrowCache := narrowcache.NewConfig(kcfg,
+		narrowcache.ConfigMaps,
+		// narrowcache.ConsolePlugin,
+		narrowcache.Daemonsets,
+		narrowcache.Deployments,
+		narrowcache.HorizontalPodAutoscalers,
+		narrowcache.Namespaces,
+		narrowcache.NetworkPolicies,
+		narrowcache.Pods,
```

Comment:
do we get/watch any Pod at some point?

Reply:
no, it's actually the `PodTemplate` I saw in pprof which comes from Deployments so this is never called. Gonna remove it.

Thanks